doc/api/assert.markdown

```diff
@@ -2,7 +2,7 @@
 
     Stability: 2 - Stable
 
-This module is used for writing unit tests for your applications, you can
+This module is used for testing actual values against expected values, you can
```

Comment:
While we're at it, can you fix the comma splice?  `This modules is used for testing actual values against expected values. You can...`